### DISCUSSION TITLE
GLMsingle version 1.2 with added diagnostics and fixes in Python

### DISCUSSION THREAD

**Discussion by iancharest** [cvnlab/GLMsingle#117] [General]
In this release of GLMsingle, we added novel diagnostics printed in the terminal, novel diagnostic figures, and we made significant efforts to make the outputs almost identical between matlab and python (up to numerical precision), for the results and figures.  

<hr /><em>This discussion was created from the release <a href='https://github.com/cvnlab/GLMsingle/releases/tag/1.2'>GLMsingle version 1.2 with added diagnostics and fixes in Python</a>.</em>

**Comment by kendrickkay** [comment]
![image](https://github.com/cvnlab/GLMsingle/assets/2728338/22cc151e-347f-4cb9-8460-0a9f7dae9b4a)
Just some eye candy: We numerically compared summary measures (in this case typeD R2 values) across Matlab and Python implementations on a diverse set of 10 datasets.  It's pretty close.
